src/constants/default-image.js

```diff
@@ -0,0 +1,3 @@
+const defaultImage = 'http://www.tcdl.io.s3-website-eu-west-1.amazonaws.com/isearch/images/thomascook_fallback-image.jpg';
```

Comment:
@jackcarlisle would you mind updating it to: https://isearch-ui.s3.amazonaws.com/thomascook_fallback-image.jpg ?